releases/release-1.25/README.md

```diff
@@ -0,0 +1,118 @@
+---
+title: "Kubernetes 1.25 Release Information"
+weight: 98
+slug: "release"
+aliases: [ "/release" ]
+description: |
+  Information regarding the current release cycle including important dates,
+  Release Team contact information, tracking spreadsheets and more!
+---
+
+# Kubernetes 1.25
+
+#### Links
+
+* [This document](https://git.k8s.io/sig-release/releases/release-1.25/README.md)
+* [Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.25/release-team.md)
+* [Meeting Minutes](http://bit.ly/k8s125-releasemtg) (members of [release-team@] receive meeting invites)
+* [v1.25 Release Calendar][k8s125-calendar]
+* Contact: [#sig-release] on slack, [release-team](mailto://release-team@kubernetes.io) on e-mail
+* [Internal Contact Info] (accessible only to members of [release-team@])
+
+#### Tracking docs
+
+* [Enhancements Tracking Sheet](https://bit.ly/k8s125-enhancements)
+* [Feature blog Tracking Sheet](http://bit.ly/k8s125-feature-blog)
+* [Bug Triage Tracking Sheet](https://bit.ly/k8s125-bug-triage-tracking)
+* [CI Signal Project Board & Issue Tracking](https://github.com/orgs/kubernetes/projects/68)
+* [Retrospective Document][Retrospective Document]
+* [kubernetes/sig-release v1.25 milestone](https://github.com/kubernetes/kubernetes/milestone/58)
+
+#### Guides
+
+* [Targeting Issues and PRs to This Milestone](https://git.k8s.io/community/contributors/devel/sig-release/release.md)
+* [Triaging and Escalating Test Failures](https://git.k8s.io/community/contributors/devel/sig-testing/testing.md#troubleshooting-a-failure)
+
+## Summary
+
+The 1.25 release cycle is as follows:
+
+- **Monday 9th May 2022**: Week 1 — Release cycle begins
```

Comment:
In the cadence KEP we proposed to have a break of 2 weeks between releases: https://github.com/kubernetes/enhancements/tree/master/keps/sig-release/2572-release-cadence#proposal

> This KEP proposes a transition to a 3-releases-per-calendar-year cadence, beginning with the Kubernetes 1.22 Release. This would result in a 15 week release cycle, with 2 weeks between release cycles. 

My question would be now: Can we start after KubeCon if we consider this time as a break anyways? Specifically May 23rd?

Reply:
If we start on May 23rd, then ~15 weeks later would be week of August 31st. 

A two week break between that and 1.26 means 1.26 would probably start around 19th of September. That would probably put us at the end of December if we held to 15 weeks. So I think we need to probably need to adjust a little bit either way? 12 weeks would end week of December 5th, 13 weeks would end week of December 12th, but also probably a lost week in there for KubeCon in October and a week for Thanksgiving in the US. Going much past the week of the 12th will run into a variety of holidays and end of the year.

Reply:
> v1.26 is scheduled to release on Dec 20, 21, or 22 (15 week cycle) or Dec 27, 28, or 29 (16 week cycle to account for KubeCon NA).

I really think these dates are too late in December. Many people are going to be unavailable and I think it's a bit unfair to expect RT folks and RelEng folks to commit to being available during these timeframes....ESPECIALLY if we end up delaying at all. I think that Dec 13/14/15 would be the absolute latest we would want to release (imo). So if we bump out the start, I think we need to have a "short" release for that last one.

Reply:
@jeremyrickard I agree that any date after Dec 15 is too late

Although this PR is to set dates for the v1.25 release, I would also like to be cognizant of the v1.26 release.
The v1.25 release cycle can align with the Release Cadence KEP with the following dates:
- v1.25 starts on May 23
- v1.25 is scheduled to be released on Aug 30 for a 15 week cycle

Then the v1.26 release cycle may need a compromise like having a 1 week break after v1.25:
- v1.26 starts on Sept 6 (day after US Labor day holiday)
- v1.26 is scheduled for release on Dec 13, 14, or 15 (15 week cycle)

Reply:
@reylejano your latest proposal looks good to me!

Reply:
We certainly don't want to delay 1.26 too much. My concern is should we save some buffer for the release in case unexpected blocker coming up?  Out of this concern, I would prefer the current schedule of `1.25 starts on May 9th and released on 16th August for a 15 week cycle`. Otherwise we would have to stick to the schedule very strict in case of delaying 1.26.

Reply:
@cici37 which benefit do we gain over the first release week starting May 9th over May 23 when we consider KubeCon (or the "time around KubeCon") as non-development frame?

@reylejano can we shrink the release cycle to still start May 23 and end earlier, e.g. August 16 (14 week cycle)? This could be done by removing the alpha.3 release for example.

Reply:
I assume it would depends on how we shrink it. We might end up losing one week of implementation or one week of releasing?(I don't think we could shrink one week of releasing time since we sure wanna things sink for some time. BTW, as @jeremyrickard mentioned earlier, there is no such thing like a "start date" of the enhancement reviewing, people could start the enhancement proposal and get it reviewed now for 1.25.

I am ok with 1.25 starts on May 23 as well.

Reply:
@saschagrunert @cici37 I think shrinking the release cycle to 14 weeks is fine
As Sascha suggested, burndown can start a week earlier at week 10, meaning that certain efforts associated with [burndown](https://github.com/kubernetes/sig-release/blob/master/releases/release_phases.md#burndown) will start a week earlier

Reply:
@saschagrunert @helayoty @reylejano @jeremyrickard I have updated to a 14 weeks release starts at May 23rd. Please have a look when have time^

Reply:
Updated based on above discussion^